### DISCUSSION TITLE
Recommend to hide issues/MRs from apps/bots by default when searching

### DISCUSSION THREAD

**Discussion by Woodz** [community/community#116904] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

When searching for issues/MRs, many repos return an overwhelming number of results from bots/apps (e.g. dependabot). Most of the time, these results are not useful so I have to manually exclude by author but this is quite tedious. I recommend adding a new filter `authored by human` or similar that is enabled by default to make it easier to search.

Example:
https://github.com/prettier/prettier/issues?q=typescript+parameter+decorator+comment returns 93 results, the vast majority of which are dependabot MRs to upgrade

**Comment by look** [comment]
Thanks for the suggestion. 

Currently, you can exclude the bots one by one like this: [typescript parameter decorator comment -author:renovate[bot] -author:dependabot[bot] -author:dependabot-preview[bot]](https://github.com/prettier/prettier/issues?q=typescript%20parameter%20decorator%20comment%20-author%3Arenovate%5Bbot%5D%20-author%3Adependabot%5Bbot%5D%20-author%3Adependabot-preview%5Bbot%5D)
